deps/src/main/java/com/microsoft/azure/sdk/iot/deps/auth/IotHubSSLContext.java

```diff
@@ -280,16 +280,8 @@ private TrustManagerFactory generateTrustManagerFactory(IotHubCertificateManager
 
     private char[] generateTemporaryPassword()
     {
-        byte[] randomBytes = new byte[256];
```

Comment:
The [ASCII table](https://www.asciitable.com/) shows that many of the characters on the lower end of the 0-255 range are inappropriate for keystore passwords